src/account-faucet.ts

```diff
@@ -1,46 +1,74 @@
 import { contractAddresses } from './contracts/addresses'
 import { interfaces } from './contracts/hardhat'
 import { secp256k1, Transaction, TransactionBody } from '@vechain/sdk-core'
-import faucetAccounts from './faucet-accounts.json'
+import { testEnv } from './test-env'
 import { ThorWallet } from './wallet'
 import { unitsUtils } from '@vechain/sdk-core'
 
 const FAUCET_AMOUNT = unitsUtils.parseVET((10_000).toString())
 const FAUCET_AMOUNT_HEX = `0x${FAUCET_AMOUNT.toString(16)}`
 
+// The funding amounts are not scaled, so `0x1` equals 1 wei
+type FundingAmounts = {
```

Comment:
We should be able to remove `FAUCET_AMOUNT` AND `FAUCET_AMOUNT_HEX` since we use specific amounts now